usaspending_api/search/v2/views/spending_by_category_views/spending_by_locations.py

```diff
@@ -103,6 +158,12 @@ def _key_to_geo_code(key):
                     "amount": int(bucket.get("sum_field", {"value": 0})["value"]) / Decimal("100"),
                 }
             )
+
+        # Combine all dicts in the `results` list that have the same values for
+        #   `code` and `name`. Sum their `amount` values together.
```

Comment:
Can you elaborate a little more on the "why" in this comment? Something about grouping results that don't match an entry in the reference table (null `code` and `name`) into a single result.